web/packages/core/src/ruffle-player.ts

```diff
@@ -1277,7 +1293,7 @@ export class RufflePlayer extends HTMLElement {
                     <p>If you are the server administrator, we invite you to try loading the file on a blank page.</p>
                 `;
                 errorFooter = `
-                    <li><a target="_top" href="${issueLink}">Report Bug</a></li>
+                    <li>${issueLink}</li>
```

Comment:
As far as I undersand, displaying "Update Ruffle" in this case doesn't make sense, as it's likely a problem with the embedding site and not Ruffle.

Reply:
We do have "polyfills" to circumvent some of these issues:
https://github.com/ruffle-rs/ruffle/blob/master/web/packages/core/src/js-polyfills.ts

So upgrading could help depending on the situation.

I also addressed your comments. I used `actionTag` instead of `actionURL` and `actionText` because imho that's shorther and a bit more flexible - but I can still change it if you want.